project/leonhsi/README.md

```diff
@@ -0,0 +1,47 @@
+# Bounding Volume Hierarchy Trees Generator
+
+###### tags: `Ray Tracing`
+###### links: [link to github repo](https://github.com/leonhsi/BVH_Generator)
+
+## Basic Information
+
+Bounding Volume Hierarchies (BVH) trees are an acceleration sturcture for ray tracing algorithm. The problem of ray tracing is to find the nearest intersection with a given ray and scene. In practice, it has to evaluate billions of such operations as the scene consists of millions of primitives, thus scene primitives are commonly arranged in spatial data structures to accelerate the search. BVH are such data structure that used mostly nowadays.
+
+In this project, I would evaluate each trees with common scenes in computer graphics such as dragon, rabbit, etc. Each of these scenes contains millions of primitives.
+
+## Problem to Solve
+There are many kinds of BVH trees, mostly divided into binary BVH and n-ary BVH, and each BVH trees has its own advantage. In this project, I want to make the process of building different BVH trees faster and eaiser by providing APIs, and evaluate their performance against each other.
+
+## Perspective Users
+Everyone that interested in computer graphics.
+
+## System Architecture
+There will be only one API and user only need to input which kinds of BVH trees he or she wants, then the output will be the performance of this BVH tree.
```

Comment:
A single API won't be very useful and it won't be easy to write Python unit testing.  I suggest to start to write tests early to give yourself a taste about it.